cluster/userdata-worker.yaml

```diff
@@ -102,7 +138,28 @@ coreos:
         Type=oneshot
         ExecStart=/usr/bin/docker pull registry.opensource.zalan.do/teapot/microscope:v0.0.2
 
+
+ 
 write_files:
+
+  - path: "/etc/profile.d/docker-path.sh"
+    content: |
+      PATH="/opt/docker:$PATH"
```

Comment:
@szuecs Ok, but the file below that seem to do the same thing is sourced in the service with `EnvironmentFile=/etc/profile.env`, so I assume this one isn't needed.